harmoni_detectors/harmoni_stt/test/deepspeech.test

```diff
@@ -0,0 +1,9 @@
+<launch>
+    <rosparam file="$(find harmoni_stt)/config/deepspeech_configuration.yaml"/>
+	<param name="instance_id" value="default"/>
+	<node pkg="harmoni_stt" type="deepspeech_service.py" name="harmoni_stt_deepspeech_default" output="screen"/>
+
+
+	<param name="test_deepspeech_input" value="$(find harmoni_stt)/test_data/hello.wav"/>
```

Comment:
The test succeeded. The output was "oh" instead of "hello", though.

Reply:
I recorded a new test audio file and updated the tests to have the correct "hello" output.